src/app/containers/ATIAnalytics/params/article/index.test.jsx

```diff
@@ -48,79 +58,23 @@ describe('ArticleAtiParams', () => {
     previousPath: '/previous-path',
     service: 'news',
     statusCode: 200,
+    atiAnalyticsAppName: 'news',
+    atiAnalyticsProducerId: 0,
   };
 
   describe('atiPageViewParams is called ', () => {
     afterEach(() => {
       jest.clearAllMocks();
     });
 
-    it('should call atiPageViewParams with the params from the Contexts', () => {
-      const mock = jest.fn().mockReturnValue('key=value&key2=value2');
-      atiPageViewParams.default = mock;
-
+    it('should call buildArticleATIUrl exactly once', () => {
```

Comment:
we should add a test to make it returns what is expected. I think this is more valuable than checking that it calls a certain function

Reply:
Done!